app/assets/stylesheets/application.css

```diff
@@ -1,15 +1,281 @@
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+:root {
+  --blue: #3778c2;
+  --green: #5bf523;
+  --gray: #737579;
+  --title-font: 'Courgette', cursive;
+  --main-font: 'Montserrat', sans-serif;
+}
+
+html {
+  box-sizing: border-box;
+  font-family: var(--main-font);
+}
+
+*,
+*::before,
+**::after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  max-width: 90%;
+}
+
+.home-container {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: whitesmoke;
+  color: var(--gray);
+  position: relative;
+}
+
+.login-link {
+  background-color: var(--blue);
+  text-decoration: none;
+  color: white;
+  font-weight: 300;
+  text-transform: uppercase;
+  display: block;
+  width: 275px;
+  padding: 17px;
+  border-radius: 5px;
+  text-align: center;
+}
+
+.signup-link {
+  text-decoration: none;
+  color: var(--gray);
+  display: block;
+  width: 275px;
+  padding: 17px;
+  text-align: center;
+}
+
+.home-link {
+  border: none;
+  background-color: transparent;
+  color: white;
+  font-size: 20px;
+  font-family: var(--main-font);
+  font-weight: 500;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.home-link:hover {
+  color: var(--green);
+}
+
+.home-links-container {
+  position: absolute;
+  bottom: 35px;
+}
+
+.main-title {
+  font-family: var(--title-font);
+  font-size: 33px;
+}
+
+.header {
+  background-color: var(--blue);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 20px;
+  color: white;
+}
+
+.header-title {
+  font-weight: 300;
+  text-transform: uppercase;
+  letter-spacing: 1.1px;
+}
+
+.nav-link {
+  border: none;
+  background-color: transparent;
+  color: white;
+  font-size: 17px;
+  font-weight: 200;
+  cursor: pointer;
+}
+
+.nav-link:hover {
+  color: var(--green);
+}
+
+.radio-buttons-container {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding-top: 10px;
+}
+
+.gray-background {
+  background-color: #e5e5e5;
+  min-height: calc(100vh - 155px);
+}
+
+.gray-background-large {
+  background-color: #e5e5e5;
+  min-height: calc(100vh - 70px);
+}
+
+.form-container {
+  background-color: white;
+  padding: 10px;
+}
+
+.input {
+  border: none;
+  width: 100%;
+  height: 20px;
+  padding: 10px 0;
+}
+
+.line {
+  border: 0.7px solid #f4f4f4;
+  margin: 0;
+}
+
+.gray-link {
+  text-decoration: none;
+  color: var(--gray);
+}
+
+.gray-links-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.green-button-container {
+  /* position: fixed;
+  bottom: 0; */
+  background-color: var(--green);
+  width: 100%;
+  padding: 20px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] Please remove the commented code here as it makes your codebase look cluttered and untidy
- [ ] Also kindly note that the background color of the button is not exactly as depicted in the Behance design. Please take a look at the attached images and make the necessary adjustments according to specifications 👍 
### Behance Button Design
![image](https://github.com/DavidEspinoG/microverse_rails_capstone/assets/46839548/384e3aab-ce10-4e25-91b9-7dc31aade899)
### Your Design
![image](https://github.com/DavidEspinoG/microverse_rails_capstone/assets/46839548/67fed888-eb00-4696-aa84-76dfb9b63fee)